cluster/ci/monitoring/prometheus_operator_deploy.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     apps.kubernetes.io/component: controller
     apps.kubernetes.io/name: prometheus-operator
-    apps.kubernetes.io/version: v0.23.2
+    apps.kubernetes.io/version: v0.29.0
```

Comment:
Can we get away with running the same version on api.ci as it has now, to reduce churn?

Reply:
Nope, because of the runc CVE last year, all containers need at least enough memory to put runc into it (about 15m). The  config-reloaded container generated by this version of the prom operator gives too little, making the alertmanager and prometheus pods unable to start.

v0.29.0 is the oldest prom operator version that has the fix for this.

xref https://bugzilla.redhat.com/show_bug.cgi?id=1735691